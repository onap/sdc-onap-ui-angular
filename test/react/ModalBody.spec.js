import React from 'react';
import ModalBody from '../../src/react/ModalBody.js';

import renderer from 'react-test-renderer';

describe('ModalBody', () => {
	test('basic test', () => {
	    const header = renderer.create(<ModalBody/>).toJSON();        
	    expect(header).toMatchSnapshot();
	});
});