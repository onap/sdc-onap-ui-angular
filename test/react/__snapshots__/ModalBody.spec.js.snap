// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalBody basic test 1`] = `
<div
  className="sdc-modal__content "
/>
`;
